data/fieldlist_NCAR.jsonc

```diff
@@ -189,7 +199,7 @@
       "standard_name": "air_temperature",
       "units": "K",
       "ndim": 4
-    },
+    // },
```

Comment:
@drewmresnick Un-comment this closing bracket--it's the cause of the json parsing error in the latest CI run.

Reply:
@drewmresnick The missing NCAR key error still seems to occur because of fieldlist conflicts.  TREFHT and T250 have the same long names,. Changing the T250 long name to "temperature_at_250_mbar_pressure_surface" (I used the LENS2 [CESM variable output list](https://www.cesm.ucar.edu/projects/community-projects/LENS2/variable-list/) as a guide) or something similar should hopefully fix this issue. I'll see if I can replicate the issue on my machine.

Reply:
Okay I did what you suggested, and it seems to be working now.